buildlib/azure-pipelines.yml

```diff
@@ -8,10 +8,16 @@ trigger:
   - master
   - v*.*.x
 
+variables:
+  DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local
+
 resources:
   containers:
-    - container: centos7_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5-cuda11:2
+    - container: centos7_cuda11_x86_64
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/centos7-mofed5-cuda11:3
+      options: $(DOCKER_OPT_VOLUMES)
+    - container: centos8_cuda11_aarch64
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/aarch64/centos8-mofed5-cuda11:3
```

Comment:
do we need DOCKER_OPT_VOLUMES here as well?

Reply:
Negative. The volume mount is required to access modules.
Modules are only used during the 'publish' stage which runs on x86 container only.